records_mover/session.py

```diff
@@ -77,6 +78,12 @@ def _infer_default_aws_creds_name(session_type: str) -> Optional[str]:
     return None
 
 
+def _infer_default_gcp_creds_name(session_type: str) -> Optional[str]:
+    if session_type == 'airflow':
+        return 'google_cloud_default'
+    return None
```

Comment:
I tried to keep the AWS and GCP creds fetching to be as similar as possible here.